pkgs/tools/misc/terminal_tarot/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "terminal_tarot";
+  version = "v0.1.1";
```

Comment:
We don't typically put the v into the version. You can just put it in the rev part...